rabbitmq/init.sls

```diff
@@ -2,10 +2,10 @@ rabbitmq-server:
   pkgrepo.managed:
     - name: deb http://www.rabbitmq.com/debian/ testing main
     - key_url: http://www.rabbitmq.com/rabbitmq-signing-key-public.asc
+    - require_in:
+      - pkg: rabbitmq-server
   pkg:
-    - installed
-    - require:
-      - pkgrepo: rabbitmq-server
+    - latest
```

Comment:
That falls under a block which starts with `if repo.startswith('ppa:'):` which is not true in this case.